### DISCUSSION TITLE
Debug messaging process

### DISCUSSION THREAD

**Discussion by sangrecita** [nextgenhealthcare/connect#6094] [Q&A] [answered]
Hello! 
I am looking for comments or suggestions or for your experience in the following context:

I have 1 channel that processes hl7 messaging, in this case Mirth adopts a bridge role, so there are no internal transformations. 
And a few days ago there was a case where a message came in at "7:45 am" and was sent at "7:50", those 5 minutes that it took to send the message I can't figure out why.

If you can guide me if there is a way that has worked for you to monitor when these situations occur.

I am currently with version 4.3.0

![image](https://github.com/nextgenhealthcare/connect/assets/146788728/913d1b42-2b55-4f49-8db9-9a2dcad83653)

**Comment by kirbykn2** [comment, marked as answer]
Check your queue configuration on the destination of the sending channel and the source on the receiving channel.

**Reply by sangrecita** [reply]
source configuration:
![image](https://github.com/nextgenhealthcare/connect/assets/146788728/46cb71bc-b909-4eb3-a8bd-8ceb4a1b25f7)

Destinations configuration:
![image](https://github.com/nextgenhealthcare/connect/assets/146788728/fd2dd1d0-d185-4690-8777-ae5d1e8ab08c)

Any suggestions? or the default is ok? Thank you
